assets/js/klarna-checkout-for-woocommerce.js

```diff
@@ -231,14 +231,15 @@ jQuery( function ( $ ) {
 						if ( name === "shipping_email" ) {
 							kco_wc.shippingEmailExists = true
 						}
-						$( "p#" + name + "_field" ).appendTo( "#kco-extra-checkout-fields" )
 					} else {
 						$( 'input[name="' + name + '"]' )
 							.closest( "p" )
 							.appendTo( "#kco-extra-checkout-fields" )
 					}
 				}
 			}
+			// Move the "Create an account?" form.
+			$( ".woocommerce-account-fields" ).appendTo( "#kco-extra-checkout-fields" )
```

Comment:
A quick thing that came up during my own testing. The checkbox `Create an account?` could have other fields that are inserted after it. So if the checkbox is checked, the account fields are not shown directly bellow it.

We could change this instead to be `$(".woocommerce-account-fields").insertAfter( $("#createaccount").closest('p') )`, that way its always inserted after the `createaccount` checkbox p wrapper.